external/emulation/emulation/zhao_carr.py

```diff
@@ -90,10 +95,36 @@ def mask_where_fortran_cloud_identical(state, emulator):
         state[Input.cloud_water],
         emulator[GscondOutput.cloud_water],
     )
-    net_condensation = cloud_out - state[Input.cloud_water]
-    return {**emulator, **apply_condensation_liquid_phase(state, net_condensation)}
+    return _update_with_net_condensation(cloud_out, state, emulator)
+
+
+def _get_classify_output(emulator, class_name):
```

Comment:
One disadvantage of passing the class_name separately is that this routine will not advertise what class names are available. What do you think about refactoring to `_get_classify_output(emulator) -> Mapping[str, tf.Tensor` where the `str` keys are the class names? This would make it compatible with the ["routing" code](https://github.com/ai2cm/fv3net/pull/1898/commits/3abd6d4254052f3f3e4547f0b60ca308e4ac65e4) I have in my PR.

Reply:
I switched it to output all available classes with names as keys.